charts/neo4j-internal/values.yaml

```diff
@@ -6,6 +6,15 @@ neo4j-standalone:
       cpu: "0.5"
       memory: "2Gi"
 
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 7474
+      runAsGroup: 7474
+      fsGroup: 7474
+      capabilities:
+        drop: ['ALL']
+      allowPrivilegeEscalation: false
+
```

Comment:
I wasn't able to test this via preview env, 
preview env only run with neo4j v5 it seems, you confirm ?

Reply:
yes but if you have identical changes, it should be fine.